### DISCUSSION TITLE
has fix for https://github.com/dotnet/maui/issues/7727 been released in 17.4?

### DISCUSSION THREAD

**Discussion by lanyusan** [dotnet/maui#11304] [General]
I have found out a control with `IsVisible = false` still takes up blank space in flex layout for windows.

According to #7727 it has been fixed. Has it been released?

This is really a big problem.

**Comment by hartez** [comment]
#7727 is referring to a bug on iOS, not Windows. And the bug was about what happens when a View's visibility changes, and wasn't specifically about FlexLayout. 

So you might be seeing a completely different bug. I'd suggest [opening an issue](https://github.com/dotnet/maui/issues/new/choose).

**Reply by lanyusan** [reply]
I thought it might be a general layout rendering problem that would have been fixed all together with #7727. Would open an issue.
